backend/src/main/scala/com/softwaremill/adopttapir/starter/formatting/GeneratedFilesFormatter.scala

```diff
@@ -62,3 +43,35 @@ final case class GeneratedFilesFormatter(filesManager: FilesManager) extends Str
     }
 
   private def isScalaFilePath(path: String): Boolean = ScalaFileExtensions.exists(e => path.endsWith(e))
+
+object GeneratedFilesFormatter extends FLogging:
+
+  private def scalafmtReporter(using CorrelationId): Resource[IO, ScalafmtReporter] =
```

Comment:
Scalafmt is now initialized as a resource. As far as I checked it should solve the [issue](https://github.com/softwaremill/adopt-tapir/issues/124).

 If there are any problems during initialization it should cause a failure of the whole application, which should then  be restarted by kubernetes.